course/api/views.py

```diff
@@ -577,3 +578,40 @@ def get_queryset(self) -> QuerySet[News]:
                     Q(audience=AUDIENCE.INTERNAL_USERS)
                 )
         return queryset
+
+
+class CourseStatisticsView(BaseStatisticsView):
+    """
+    Returns submission statistics for a course, over a given time window.
+
+    Returns the following attributes:
+
+    - `submission_count`: total number of submissions.
+    - `submitters`: number of users submitting.
+
+    Operations
+    ----------
+
+    `GET /courses/<course_id>/statistics/`:
+        returns the statistics for the given course.
+
+    - URL parameters:
+        - `endtime`: date and time in ISO 8601 format indicating the end point
+          of time window we are interested in. Default: now.
+        - `starttime`: date and time in ISO 8601 format indicating the start point
+          of time window we are interested in. Default: one day before endtime
+    """
+
+    serializer_class = CourseStatisticsSerializer
+
+    def get_queryset(self) -> QuerySet:
+        queryset = super().get_queryset()
+        course_id = self.kwargs['course_id']
+        return queryset.filter(
+            exercise__course_module__course_instance=course_id
```

Comment:
```suggestion
            exercise__course_module__course_instance=course_id,
```
The trailing comma is used in this "one param per line" format so that you can later add new parameters without adding changes (git diff) to the previous line that is otherwise not going to be changed.